src/yandex_cloud_ml_sdk/_models/completions/model.py

```diff
@@ -119,12 +134,19 @@ def _make_request(
 
         return CompletionRequest(
             model_uri=self._uri,
-            completion_options=CompletionOptions(**completion_options_kwargs),
+            completion_options=self._make_completion_options(stream=stream),
             messages=messages_to_proto(messages),
             tools=[tool._to_proto(ProtoCompletionsTool) for tool in tools],
             **response_format_kwargs,
         )
 
+    def _make_batch_request(self, dataset_id: str) -> BatchCompletionRequest:
+        return BatchCompletionRequest(
+            model_uri=self.uri,
+            completion_options=self._make_completion_options(stream=False),
```

Comment:
There is no stream option in batch completion options
https://a.yandex-team.ru/cloudia/cloud/fomo/tuning-services/batch-inference-service-parent/fomo-batch-inference-api/src/main/proto/yandex/cloud/priv/ai/batch_inference/v1/inference_options.proto?rev=r58077#L26